test/multiverse/suites/unicorn/Envfile

```diff
@@ -0,0 +1,17 @@
+SUPPORTED_UNICORN_VERSIONS = [
+  nil,
+  ", '6.1.0'",
+  ", '4.9.0'"
+]
+
+SUPPORTED_UNICORN_VERSIONS << ", '5.3.0'" if RUBY_VERSION < '3.0.0'
+
+SUPPORTED_UNICORN_VERSIONS.each do |version|
+  next if RUBY_PLATFORM == 'java'
```

Comment:
It'd be nice to be able to declare the "JRuby doesn't get tested here at all" bit in a higher level way for Envfiles. As-is, the array gets defined, appended to, and then iterated over with fully all 4 supported versions being examined.

Perhaps for now (shy of an Envfile overhaul) we could do one of these things:

- Change the instantiation of `SUPPORTED_UNICORN_VERSIONS` and the appending to it to result in `[]` as the value for JRuby.
- Have `break` hit once instead of `next` hit four times.

Reply:
Ah, great catch. While investigating the Envfile refactor, I found this option that already exists to skip JRuby: 

```ruby
# test/multiverse/suites/bunny/Envfile

suite_condition("bunny not supported for JRuby") do
  RUBY_PLATFORM != 'java'
end
```

I forgot to mention this during our last meeting about Envfiles. I'd like to integrate it into all our suites that skip JRuby. Would this work for you?

Reply:
Yep, that sounds great. I was thinking you'd have a great long term fix like that as part of the Envfile overhaul work. That's even better than the short-term stopgap measures I was proposing, definitely.